Resources/Prototypes/_Sunrise/Entities/Objects/Weapons/Guns/Ammunition/Cartridges/heavy_rifle_nt.yml

```
@@ -0,0 +1,99 @@
+- type: entity
+  id: BaseCartridgeHeavyRifleNT
+  name: cartridge (7,62mm NT) # 7.62x51 NATO > 7.62x51 NT
+  parent: [ BaseCartridge, BaseRestrictedContraband ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)